car.capnp

```diff
@@ -456,8 +457,9 @@ struct CarParams {
 
   enum TransmissionType {
     unknown @0;
-    automatic @1;
-    manual @2;
+    automatic @1;  # Traditional auto, including DSG
+    manual @2;  # True "stick shift" only
+    direct @3;  # Electric vehicle or other direct drive
```

Comment:
is the distinction between "direct" and "automatic" relevant in the upcoming openpilot changes? If not, I think automatic could just include "direct" transmissions too.

Reply:
Yes, that's why I need it. For auto MQBs except e-Golf, including hybrids, we get gearshift position from a message sent by the auto transmission controller. For the e-Golf, there isn't one, so we have to fish it out of an EV-specific message.

e-Golf: EV_Gearshift.GearPosition
Manual trans: ( synthesized from ECM clutch-pressed switch and BCM reverse light )
Everything else: Getriebe_11.GE_Fahrstufe